Makefile

```diff
@@ -7,10 +8,17 @@ TOOLCHAIN = +$(RUST_VERSION)
 BUILD_ARG = $(TOOLCHAIN) build -r
 BUILD_DIR = ./target/release
 BUNDLE_DIR = bundle
+
+ifeq ($(DETECTED_OS),windows)
+ZSTD_LIB_ARGS = -j lib-nomt UNAME=Windows ZSTD_LIB_COMPRESSION=0 ZSTD_LIB_DICTBUILDER=0 AR="zig ar"
+ZSTD_LIB_CC_ARGS = -s -O3 -flto
+ZSTD_LIB_CC_x64 = CC="zig cc -target x86_64-windows-gnu $(ZSTD_LIB_CC_ARGS)"
+else
 ZSTD_LIB_ARGS = -j lib-nomt UNAME=Linux ZSTD_LIB_COMPRESSION=0 ZSTD_LIB_DICTBUILDER=0 AR="zig ar"
 ZSTD_LIB_CC_ARGS = -s -O3 -flto
 ZSTD_LIB_CC_arm64 = CC="zig cc -target aarch64-linux-musl $(ZSTD_LIB_CC_ARGS)"
 ZSTD_LIB_CC_x64 = CC="zig cc -target x86_64-linux-musl $(ZSTD_LIB_CC_ARGS)"
+endif
```

Comment:
This is a Lambda only optimization where we pack LLRT into a self extrating zstandard compressed executable. These args should not differ on windows as we should be able to build LLRT for Linux x64 and ARM64 on Windows (theoretically)

Reply:
I am not familiar with zstd related compilation parameters. After testing, ZSTD_LIB_CC_x64 must be distinguished, otherwise ```error: the argument -femit-implib is allowed only when building a Windows DLL```